ansible/configs/sap-integration/post_software.yml

```diff
@@ -0,0 +1,371 @@
+      delay: 5
+      until: getroute is succeeded
+
+    - name: Test that route is reachable
+      uri:
+        url: "http://{{ getroute.stdout|from_json|json_query('items[0].spec.host') }}"
+      register: testroute
+      retries: 15
+      delay: 5
+      until: testroute is succeeded
+
+    - name: Delete project
+      command: oc delete project postflightcheck
+
+    - name: Switch back to default project
+      command: oc project default
+
+    - debug:
+        msg: "{{ item }}"
+      loop:
+      - "user.info: "
+      - "user.info: Post Flight Check"
+      - "user.info: DNS Web Console ............... {{ 'OK' if checkdnswebconsole.rc == 0 else 'FAIL' }}"
+      - "user.info: DNS API ....................... {{ 'OK' if checkdnsapi.rc == 0 else 'FAIL' }}"
+      - "user.info: Web console ................... {{ 'OK' if testwebconsole is succeeded else 'FAIL' }}"
+      - "user.info: API ........................... {{ 'OK' if clusterinfor.rc == 0 else 'FAIL' }}"
+      - "user.info: Create Project with PV ........ {{ 'OK' if newproject.rc == 0 else 'FAIL' }}"
+      - "user.info: App deployed .................. {{ 'OK' if phpw.rc == 0 and mysqlw.rc == 0 else 'FAIL' }}"
+      - "user.info: Route ......................... {{ 'OK' if testroute is succeeded else 'FAIL' }}"
+
+    - name: Fail smoke tests
+      when:
+      - run_smoke_tests | default( False ) | bool
+      - >-
+        checkdnswebconsole.rc != 0
+        or checkdnsapi.rc != 0
+        or testwebconsole is failed
+        or clusterinfor.rc != 0
+        or newproject.rc != 0
+        or phpw.rc != 0
+        or mysqlw.rc != 0
+        or testroute is failed
+      fail:
+        msg: "FAIL Smoke tests"
+      ignore_errors: no
+
+# Deploy Workloads
+- name: Step 005.2 - Deploy Infra and Student Workloads
+  import_playbook: workloads.yml
+
+# Workarounds to enable cluster shutdown before 24h are over
+- name: Step 005.3 - Enable Cluster Shutdown and Resume
+  hosts: bastions
+  run_once: yes
+  gather_facts: false
+  become: false
+  environment:
+    KUBECONFIG: /home/{{ ansible_user }}/{{ cluster_name }}/auth/kubeconfig
+  tasks:
+  - when:
+    - install_ocp4 | default( False ) | bool
+    - ocp4_enable_cluster_shutdown | default( False ) | bool
+    block:
+    - name: Set Ansible Python interpreter to k8s virtualenv
+      set_fact:
+        ansible_python_interpreter: /opt/virtualenvs/k8s/bin/python
+    - name: Create Daemon Set to renew Bootstrap Credentials
+      k8s:
+        state: present
+        definition: "{{ lookup('file', './files/kubelet-bootstrap-cred-manager-ds.yaml' ) | from_yaml }}"
+    - name: Delete initial Bootstrap Secrets to force regeneration
+      k8s:
+        state: absent
+        api_version: v1
+        kind: Secret
+        name: "{{ item }}"
+        namespace: openshift-kube-controller-manager-operator
+      loop:
+      - "csr-signer-signer"
+      - "csr-signer"
+
+- name: Step 005.4 Run performance test (optional)
```

Comment:
This was put in place for a very specific purpose to test ceph performance, but you should not have it in your SAP config.

Reply:
done